importer/csv/locations/locations_full.csv

```diff
@@ -1,4 +1,4 @@
-name,status,method,id,parentName,parentID,type,typeCode,physicalType,physicalTypeCode,longitude,latitude
-City1,active,update,ba787982-b973-4bd5-854e-eacbe161e297,test location-1,18fcbc2e-4240-4a84-a270-7a444523d7b6,site,si,ward,wa,36.81,-1.28
-Building1,active,update,0a04f1c2-de2a-4869-bab2-763d017e5316,test location-1,18fcbc2e-4240-4a84-a270-7a444523d7b6,building,bu,building,bu,,
-City2,active,create,,test location-1,18fcbc2e-4240-4a84-a270-7a444523d7b6,jurisdiction,jdn,jurisdiction,jdn,,
\ No newline at end of file
+name,status,method,id,parentName,parentID,type,typeCode,adminLevel,physicalType,physicalTypeCode,longitude,latitude
+City1,active,update,ba787982-b973-4bd5-854e-eacbe161e297,test location-1,18fcbc2e-4240-4a84-a270-7a444523d7b6,site,si,0,ward,wa,36.81,-1.28
```

Comment:
I know this is just an example csv, but since these locations have a parent, let's not have them with an admin level that is 0. I think the actual level should be 3, but let's just check the api to confirm and update